opal/threads/wait_sync.h

```diff
@@ -75,7 +76,7 @@ typedef struct ompi_wait_sync_t {
         (sync)->signaling = false;                    \
 }
 
-OPAL_DECLSPEC int sync_wait_mt(ompi_wait_sync_t *sync);
+OPAL_DECLSPEC int opal_sync_wait_mt(ompi_wait_sync_t *sync);
 static inline int sync_wait_st (ompi_wait_sync_t *sync)
```

Comment:
Just for symmetry -- even if the `_st` symbol isn't actually a problem -- should probably make both the `_mt` and `_st` versions be prefixed.